### DISCUSSION TITLE
Произошла непредвиденная ошибка при загрузке карты

### DISCUSSION THREAD

**Discussion by VychuzhaninovNA** [cdek-it/widget#70] [Q&A] [answered]
Добрый день.
Был настроен виджет СДЭКа, все работало.
Потом в один момент появилась проблема с доступам к скриптам
	<script src="https://cdn.jsdelivr.net/npm/@cdek-it/widget@3" type="text/javascript"></script>
	<script src="https://cdn.jsdelivr.net/npm/@unocss/runtime" type="text/javascript"></script>
Скрипты сохранил и подключил со своего сервера. Все работало.
Через какое-то карта работать перестала, пишет: Произошла непредвиденная ошибка при загрузке карты
Пробовал обратно вставлять ссылки на скрипты, ключ яндекса активен, ничего не помогает. 
Пробовал исходный код примера виджета себе вставлять, просто чтобы проверить работу карты, все равно эта ошибка.
Как определить причину, почему карта не загружается?

**Comment by piton-zel** [comment, marked as answer]
F12 и смотри что пишет в консоле, я тут 2 варианта вижу, или он не дожидается загрузки точек (запрос deliverypoints), или где-то рядом на странице есть ошибка по JS.
Но мне кажется ваш случай это номер один все же.

**Reply by VychuzhaninovNA** [reply]
спасибо!
оказывается ошибка в service.php
ключи слетели, новые выдали всем, после недавних событий )
